### PR TITLE
MAGETWO-98251 Position remove icon in sidebar

### PR DESCRIPTION
**Pull request by ptylek**
### Description (*)

Remove icon that appears in Wishlist in sidebar receives line-height and width (each 35px) from listings.less what in the end changes its position. I unset line-height and set width to auto in module.less file in order to keep sidebar css changes related to product list in sidebar in one location. Other approach is to add `:not(.sidebar)` to listings.less.

### Fixed Issues (if relevant)

1. magento/magento2#21190: The Wishlist removing icon isn't correctly positioned on category pages

### Manual testing scenarios (*)

After changes:
image

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25380.instances.magento-community.engineering).
Admin access: [https://pr-25380.instances.magento-community.engineering/admin](https://pr-25380.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25380.instances.magento-community.engineering).
Admin access: [https://pr-25380.instances.magento-community.engineering/admin](https://pr-25380.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by ptylek** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [Magento instance](https://i-25380-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25380-2-3-develop.instances.magento-community.engineering/admin](https://i-25380-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by ptylek** [comment]
Fixed in Blank and Luma

File: app/design/frontend/Magento/blank/web/css/source/_layout.less [hunk 1]
```diff
@@ -125,11 +125,16 @@
     }
 
     .page-layout-2columns-left {
-        .sidebar-additional {
-            clear: left;
-            float: left;
-            padding-left: 0;
-            padding-right: @layout-column__additional-sidebar-offset;
+        .page-layout-2columns-left {
```
Comment (krzksz) [inline app/design/frontend/Magento/blank/web/css/source/_layout.less]: Is this double nesting needed here?

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less [hunk 2]
```diff
@@ -397,7 +397,7 @@
 
     .box-tocart {
         &:extend(.abs-box-tocart all);
-        
+
```
Comment (krzksz) [inline app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less:400]: I don't thing this break line should be a part of this PR.

**Review by krzksz — CHANGES_REQUESTED** [review]

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less (hunk 2 above)
Comment (ptylek) [reply app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less:400]: Line 400 has new line in the original file. Not sure why it appears like this as diff.
image

File: app/design/frontend/Magento/blank/web/css/source/_layout.less (hunk 1 above)
Comment (ptylek) [reply app/design/frontend/Magento/blank/web/css/source/_layout.less]: Thanks! Good catch. Fixed as requested.

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less (hunk 2 above)
Comment (krzksz) [reply app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less:400]: Alright.

File: app/design/frontend/Magento/blank/web/css/source/_layout.less (hunk 1 above)
Comment (krzksz) [reply app/design/frontend/Magento/blank/web/css/source/_layout.less]: Thanks.

**Review by krzksz — APPROVED** [review]

**Review by aleron75 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @aleron75, thank you for the review.
[ENGCOM-6233](https://jira.corp.magento.com/browse/ENGCOM-6233) has been created to process this Pull Request
:eight_spoked_asterisk: @aleron75, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
Hi @ptylek 

During testing, we faced the issue

**_Problem:_**  On the category page, the "Add to Cart" button is torn horizontally

**Actual Result:**
after1

@ptylek Could you take a look?

Thanks!

**Comment by ptylek** [comment]
@engcom-Alfa I'll have a look

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25380.instances.magento-community.engineering).
Admin access: [https://pr-25380.instances.magento-community.engineering/admin](https://pr-25380.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25380.instances.magento-community.engineering).
Admin access: [https://pr-25380.instances.magento-community.engineering/admin](https://pr-25380.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25380.instances.magento-community.engineering).
Admin access: [https://pr-25380.instances.magento-community.engineering/admin](https://pr-25380.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by ptylek** [comment]
@engcom-Alfa I fixed that issue.

image

**Comment by aleron75** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @aleron75. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @aleron75, here is your [new Magento instance](https://pr-25380.instances.magento-community.engineering).
Admin access: [https://pr-25380.instances.magento-community.engineering/admin](https://pr-25380.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by aleron75 — APPROVED** [review]
works for me, good job

**Comment by magento-engcom-team** [comment]
Hi @aleron75, thank you for the review.
[ENGCOM-6233](https://jira.corp.magento.com/browse/ENGCOM-6233) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Review by krzksz — APPROVED** [review]

**Pull request merged**